app/json_parser.py

```diff
@@ -0,0 +1,23 @@
+import json
+from typing import Any
+
+
+def read_json_from_file(file_path: str) -> Any:
+    try:
+        with open(file_path, "r") as file:
+            json_data = json.load(file)
+        return json_data
+    except FileNotFoundError:
+        raise FileNotFoundError(f"File not fount: {file_path}")
```

Comment:
Don't catch this error here, you just raise it again with defferent error message. No need to do that